internal/entimport/mysql.go

```diff
@@ -42,7 +42,22 @@ func (m *MySQL) SchemaMutations(ctx context.Context) ([]schemast.Mutator, error)
 	if err != nil {
 		return nil, err
 	}
-	return schemaMutations(m.field, s.Tables)
+	tables := s.Tables
+	if m.excludedTables != nil {
+		tables = nil
+		excludedTableNames := make(map[string]bool)
+		for _, t := range m.excludedTables {
+			excludedTableNames[t] = true
+		}
+		// filter out tables that are in excludedTables:
+		for _, t := range s.Tables {
+			if !excludedTableNames[t.Name] {
+				tables = append(tables, t)
+			} else {
+			}
```

Comment:
Do you see the lint issue here?
```suggestion
			}
```